src/solver_specific/gurobi.jl

```diff
@@ -6,12 +6,17 @@ function run_scenario_gurobi(; solver_kwargs::Union{Dict, Nothing}=nothing, kwar
     solver_kwargs = something(solver_kwargs, Dict("Method" => 2, "Crossover" => 0))
     try
         global env = Gurobi.Env()
+    catch e
+        println("Error encountered starting Gurobi.")
```

Comment:
I'm not seeing this message printed in the snippet that you posted. Not sure if you only posted the error and stacktrace, so it might make sense that we can't see the message

Reply:
You're right, I didn't post the full thing, but it does show up and I've updated the original post.